providers/capv/standard/test_data/cluster_values.yaml

```diff
@@ -26,12 +26,13 @@ connectivity:
   bastion:
     enabled: true
   network:
+    allowAllEgress: true
     controlPlaneEndpoint:
-      host: "10.10.222.231"  # [string] Manually select an IP for kube API.
+      host: ""  # [string] Manually select an IP for kube API. "" for auto selection from the ipPoolName pool.
       port: 6443
+      ipPoolName: "wc-cp-ips"  # [string] Ip for control plane will be drawn from this GlobalInClusterIPPool.
     loadBalancers:
-      cidrBlocks:
-      - "10.10.222.232/30"
+      cidrBlocks: []
```

Comment:
Can we remove this as it's the default in the `values.yaml` ?

Reply:
Should be gone. However this is not solved yet for testing, this IP range basically tells kubevip cloud provider that's running in WC, what IPs it can allocate for Services with type LoadBalancer. We need at least one IP for the ingress controller and the rest can be done using Ingress (grafana, dex and what not). So once/if the tests start to do something smart like checking if service A is reachable by dns name, it will probably fail.

It's my next thing to work on, though.

Reply:
our schema is not happy about it:

```
k describe app t-oyeoqr6 -n org-giantswarm | grep -A4 Reason                                                                                                                                ○ gcapeverde-admin@gcapeverde
    Reason:         values don't meet the specifications of the schema(s) in the following chart(s):
cluster-vsphere:
- connectivity.network.loadBalancers.cidrBlocks: Array must have at least 1 items

    Status:  not-installed
```
so putting this back

Reply:
Can't you remove both the `controlPlaneEndpoint` and `loadBalancers` blocks as they already have the same defaults baked into the chart.

Reply:
Oh, sorry I follow now. I thought the default values had a CIDR specified but I was looking at the wrong bit.